site/docs/spark-procedures.md

```diff
@@ -334,6 +339,13 @@ CALL catalog_name.system.rewrite_manifests('db.sample', false)
 
 The `snapshot` and `migrate` procedures help test and migrate existing Hive or Spark tables to Iceberg.
 
+!!! Note
+    Parquet files written with Parquet writers that use names other than `list` and `element` for repeated group
+    and element of the list respectively are **read incorrectly as nulls** by Iceberg upto 0.12.1 Iceberg versions.
+    Most commonly such files are written by follow writers. 
```

Comment:
nit: `the following writers:`

btw, have you tried `mkdocs serve`? Not sure if the indentation would be correctly displayed, please verify that.

Reply:
Hey @jackye1995 , to ease the cherry-picks we decided to extract out format change to a separate PR. Will follow up on this right after we get this PR in.

Reply:
sure, makes sense

Reply:
Updated docs to take of nit comment and made sure it looks as expected with `mkdocs serve` as well.